modules/nw-kube-proxy-config.adoc

```diff
@@ -20,10 +23,10 @@ link:https://golang.org/pkg/time/#ParseDuration[Go time package] documentation.
 
 |`proxyArguments.iptables-min-sync-period`
 |The minimum duration before refreshing `iptables` rules. This parameter ensures
-that the refresh does not happen too frequently.
+that the refresh does not happen too frequently. By default, a refresh starts as soon as a change that affects `iptables` rules occurs.
```

Comment:
@danwinship -- I updated the description for this here.

Reply:
This looks good